dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -351,6 +352,103 @@ public async Task AzureOpenAIDefaultValueTestAsync()
         Assert.Contains("Bob", actual.Result, StringComparison.OrdinalIgnoreCase);
     }
 
+    [Fact]
+    public async Task MultipleServiceInlineServiceIdTestAsync()
+    {
+        // Arrange
+        var azureOpenAIConfiguration = this._configuration.GetSection("AzureOpenAI").Get<AzureOpenAIConfiguration>();
+        Assert.NotNull(azureOpenAIConfiguration);
```

Comment:
It looks like this check is not needed here, because we already fetch the configuration and check if it's null in two methods below: `ConfigureAzureOpenAI` and `ConfigureInvalidAzureOpenAI`.

Same applies to other test methods where this check is performed.

Reply:
Good catch, looks like the pattern was changed in some tests and this code was never removed and now I'm copying the wrong pattern